src/shims/linux_stubs.c

```diff
@@ -40,8 +40,9 @@ char* mach_error_string(mach_msg_return_t x) {
 void mach_vm_deallocate() { LINUX_PORT_ERROR();  }
 
 mach_port_t pthread_mach_thread_np(void) {
-  return (pid_t)syscall(SYS_gettid);
```

Comment:
Yes we agree, once you merge Dave's change, we will fix this. Right now we need it to get this compile and run.